src/imitation/algorithms/preference_comparisons.py

```diff
@@ -1599,8 +1597,11 @@ def train(
                 epoch_multiplier = self.initial_epoch_multiplier
 
             self.reward_trainer.train(self.dataset, epoch_multiplier=epoch_multiplier)
-            reward_loss = self.logger.name_to_value["reward/final/train/loss"]
-            reward_accuracy = self.logger.name_to_value["reward/final/train/accuracy"]
+            base_key = self.logger.get_prefixes() + "reward/final/train"
+            assert f"{base_key}/loss" in self.logger.name_to_value
```

Comment:
This seems unnecessary (if it's not present we get a KeyError in the next line anyway), is it to prevent a type warning?

Reply:
Actually if the key is not found, it defaults to zero, which is a (IMO bad) silent error.

Reply:
Oh, doh, didn't realize it was a defaultdict. Yeah, that's pretty bad.